frontend/public/style/_overrides.scss

```diff
@@ -158,6 +158,9 @@ kbd {
     font-size: $font-size-base;
     height: auto;
   }
+  .pf-l-stack {
+    font-size: $font-size-base;
```

Comment:
@spadgett didn't we talk about changing the default font size globally?
cc @sg00dwin

Reply:
> @spadgett didn't we talk about changing the default font size globally?

Yes, @sg00dwin is still looking at it. It sounds more involved than just updating one var unfortunately.

Reply:
ok for now we will go with this change then